gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_base.py.j2

```diff
@@ -66,7 +65,12 @@ class _Base{{ service.name }}RestTransport({{service.name}}Transport):
         Args:
             host ({% if service.host %}Optional[str]{% else %}str{% endif %}):
                 {{ ' ' }}The hostname to connect to {% if service.host %}(default: '{{ service.host }}'){% endif %}.
-            credentials (Optional[google.auth.credentials.Credentials]): The
+            {# Note: Type hint for credentials is set to `Any` to support
+             # async and sync credential types in the parent rest transport classes. 
+             # However, we should have a stronger type here such as an abstract base credentials
```

Comment:
Do we need a TODO comment/bug to follow up

Reply:
addressed.